src/marqo/tensor_search/on_start_script.py

```diff
@@ -117,11 +121,19 @@ class ModelsForCacheing:
 
     def __init__(self):
         import torch
-      
-        self.models = (
-            'onnx/all_datasets_v4_MiniLM-L6',
-            "ViT-L/14",
-        )
+        warmed_models = utils.read_env_vars_and_defaults(EnvVars.MARQO_MODELS_TO_PRELOAD)
+        if warmed_models is None:
+            self.models = []
+        elif isinstance(warmed_models, str):
+            try:
+                self.models = json.loads(warmed_models)
+            except json.JSONDecodeError as e:
+                raise errors.EnvVarError(
+                    f"Could not parse environment variable `{EnvVars.MARQO_MODELS_TO_PRELOAD}`. "
+                    f"Please ensure that this a JSON-encoded array of strings."
```

Comment:
Can we add another line to this giving an example of a JSON-encoded array. First time marqo users may not know what they are and therefore, this will cause friction. 

```
raise errors.EnvVarError(
    f"Could not parse environment variable `{EnvVars.MARQO_MODELS_TO_PRELOAD}`. "
    f"Please ensure that this a JSON-encoded array of strings."
    f" "export `{EnvVars.MARQO_MODELS_TO_PRELOAD}`='["ViT-L/14", "onnx/all_datasets_v4_MiniLM-L6"]'
```

Reply:
Thanks @Jeadie  - added your suggestion